test/kuttl/tests/glance_edge/02-cleanup_glance.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      oc kustomize ../../../../config/samples/layout/split | oc delete -n $NAMESPACE -f -
```

Comment:
Should this use `../../../../config/samples/layout/edge` since we used that in `01-deploy_glance.yaml`?

Reply:
oh thank you, good catch! :D